libraries/radspreadprocessing/formats-and-conversion/data-table/settings.md

```diff
@@ -60,6 +60,34 @@ The __CellImportedEventArgs__ contains information about the current cell:
 * __DataTableCulture:__ Get or sets the DataTable culture. By default the culture of the workbook is used.
 * __RangeToExport:__ Get or sets the cell range for which the data will be exported.
 
+## The ColumnExporting event  
+
+The __ColumnExporting__ event is fired for each column before the column is added to the table and allows you to change its properties.
+
+The __ColumnExportingEventArgs__ object contains the the current column instance and its index:
+* __DataColumn:__ Gets the DataColumn that is being exported.
```

Comment:
Add link to the DataColumn class - https://docs.microsoft.com/en-us/dotnet/api/system.data.datacolumn?view=net-6.0

Reply:
Added link